docs/normen/privacy en gegevensbescherming/Beginsel van eerlijkheid (Fairness).md

```diff
@@ -0,0 +1,41 @@
+---
+title: Beginsel van eerlijkheid (Fairness)
+
+---
+
+!!! info "Disclaimer"
+
+    Het Algoritmekader is nog volop in ontwikkeling. Op deze plek willen we vooral aan de slag gaan op een open en transparante wijze. Het is dus niet definitief. Dat betekent dat er dingen opstaan die niet af zijn en soms zelfs fout. Mocht er iets niet kloppen, laat het ons weten via [GitHub](https://github.com/MinBZK/Algoritmekader).
+
+
+## Norm
+**De verwerking van persoonsgegevens op een eerlijke en rechtmatige manier moet gebeuren.**
```

Comment:
De verwerking van persoonsgegevens moet op een eerlijke en rechtmatige manier gebeuren.

Reply:
Kan de zin worden aangepast naar:

> De verwerking van persoonsgegevens moet aantoonbaar op een rechtmatige, eerlijke en transparante manier gebeuren.

Dat dekt artikel 5(1)(a) en 5(2) van de AVG. Vooral 5(2) lijkt me dusdanig veel impact hebben dat het belangrijk is om die zo helder mogelijk te benoemen.

Reply:
Wat is de toegevoegde waarde van het abstract herformuleren van de AVG? Voor de uitvoeringspraktijk zou ik dan eerder op zoek gaan naar concretiseren van AVG-normen die relevant zijn, en idem voor de AI Act.

Reply:
Ik denk dat 5(2) niet zoveel uitleg nodig heeft maar wel heel duidelijk benoemd moet worden. Dat was de meest belangrijke reden voor dit voorstel.

Reply:
> Wat is de toegevoegde waarde van het abstract herformuleren van de AVG? Voor de uitvoeringspraktijk zou ik dan eerder op zoek gaan naar concretiseren van AVG-normen die relevant zijn, en idem voor de AI Act.

We zijn bezig met het vertalen van normen naar een praktische aanpak. Jullie zullen de komende weken meer in GitHub zien  ;-)

Reply:
Ik denk dat ik maar even wacht tot duidelijk is wat voor input waar gewenst is.